phone_verify/api.py

```diff
@@ -21,7 +21,8 @@ def register(self, request):
         serializer = PhoneSerializer(data=request.data)
         serializer.is_valid(raise_exception=True)
         session_token = send_security_code_and_generate_session_token(
-            str(serializer.validated_data["phone_number"])
+            str(serializer.validated_data["phone_number"]),
+            serializer.validated_data['language']
```

Comment:
I'd suggest to take the language from the value of `Accept-Language` HTTP header.